clang/labs/lab1/ivanov_nikita/deprecatedMatcher.cpp

```diff
@@ -0,0 +1,56 @@
+#include "clang/AST/ASTConsumer.h"
+#include "clang/AST/RecursiveASTVisitor.h"
+#include "clang/Frontend/CompilerInstance.h"
+#include "clang/Frontend/FrontendPluginRegistry.h"
+
+using namespace clang;
+
+class DepConsumer : public ASTConsumer {
+  CompilerInstance &Instance;
+
+public:
+    explicit DepConsumer(CompilerInstance &CI): Instance(CI) {}
```

Comment:
Overall, LGTM.
The last remark is: could you please align formatting of your files with the help of clang-format tool?